### DISCUSSION TITLE
Folder is not shown when using path navigate on Storage UI

### DISCUSSION THREAD

**Discussion by nthouliss** [supabase/supabase#16939] [Contribute to Supabase]
# Bug report

## Describe the bug

When doing a path search on the Storage UI, the folders are still shown in alphabetical order, rather than the one you searched for being at the top

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to Storage UI
2. Use the path navigate search to find a folder
3. Folder is not the top result

## Expected behavior

When I use the navigate on the Storage UI I expect that the folder I searched for will be at the top (or the only one shown)

## System information

- OS: macOS

## Additional context

I'd consider this a bug because when you have a large amount on folders it's impossible to get to the one you want, which is important to be able to select the folder to delete it in the UI. 

When navigating to a file I want it to be at the top, so I can easily interact with it. I've used folders in this example, but I believe this applies to individual files as well

**Comment by Hallidayo** [comment]
Hi @nthouliss - Have you got a video/capture of this issue? I'm trying to recreate locally but I am unable to.

**Comment by nthouliss** [comment]
@Hallidayo here's a screen capture, when I navigate to a folder it doesn't actually navigate to it (for instance show that folder at the top of the UI) and if I try to scroll down it won't keep loading (it only does it the once). This was captured on Firefox but I was able to replicate on Chrome as well. 

https://user-images.githubusercontent.com/53867736/208329077-dc1548a5-7c93-4386-b5f6-e935e5def986.mp4
